examples/list_red_moving_vehicle/getcolor.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+import numpy as np
+import cv2
```

Comment:
Could you dump your requirements file for each example?

Reply:
And if VQPy has dependencies, you can use `install_requires` argument of the `setuptools.setup` function in `setup.py` file.

Reply:
Not resolved yet. I have create an issue #7 for easy tracking